frontend/src/wizards/Governance/Policy/PolicyWizard.tsx

```diff
@@ -439,7 +439,7 @@ export function PolicyWizardTemplates(props: { policies: IResource[] }) {
                 label={t('Namespace')}
                 labelHelp={t('The operator is installed in this namespace.')}
               />
-              <WizTextInput path="objectDefinition.spec.subscription.channel" label={t('Channel')} required />
+              <WizTextInput path="objectDefinition.spec.subscription.channel" label={t('Channel')} />
```

Comment:
This shouldn't be necessary once you rebase on the latest main branch due to this PR merging:
https://github.com/stolostron/console/pull/3422